pkg/minikube/cluster/cluster.go

```diff
@@ -740,34 +616,14 @@ func GetKubernetesEndpointsWithNamespace(namespace string) (endpointGetter, erro
 
 // EnsureMinikubeRunningOrExit checks that minikube has a status available and that
 // that the status is `Running`, otherwise it will exit
-func EnsureMinikubeRunningOrExit(api libmachine.API) {
+func EnsureMinikubeRunningOrExit(api libmachine.API, exitStatus int) {
```

Comment:
I understand why you did this but it seems like a weird fn signature to me.  Maybe it could just return an error?  That would seem more normal to me. However, it might be outside the scope of this PR to refactor this.